UM/Math/Matrix.py

```diff
@@ -15,6 +15,23 @@
 # Heavily based (in most cases a straight copy with some refactoring) on the excellent
 # 'library' Transformations.py created by Christoph Gohlke.
 class Matrix(object):
+    # epsilon for testing whether a number is close to zero
+    _EPS = numpy.finfo(float).eps * 4.0
+
+    # map axes strings to/from tuples of inner axis, parity, repetition, frame
+    _AXES2TUPLE = {
+    "sxyz": (0, 0, 0, 0), "sxyx": (0, 0, 1, 0), "sxzy": (0, 1, 0, 0),
+    "sxzx": (0, 1, 1, 0), "syzx": (1, 0, 0, 0), "syzy": (1, 0, 1, 0),
+    "syxz": (1, 1, 0, 0), "syxy": (1, 1, 1, 0), "szxy": (2, 0, 0, 0),
+    "szxz": (2, 0, 1, 0), "szyx": (2, 1, 0, 0), "szyz": (2, 1, 1, 0),
+    "rzyx": (0, 0, 0, 1), "rxyx": (0, 0, 1, 1), "ryzx": (0, 1, 0, 1),
+    "rxzx": (0, 1, 1, 1), "rxzy": (1, 0, 0, 1), "ryzy": (1, 0, 1, 1),
+    "rzxy": (1, 1, 0, 1), "ryxy": (1, 1, 1, 1), "ryxz": (2, 0, 0, 1),
+    "rzxz": (2, 0, 1, 1), "rxyz": (2, 1, 0, 1), "rzyz": (2, 1, 1, 1)}
```

Comment:
In the original there was an explanation for these strings:
* First character: Rotations are applied to `s`tatic or `r`otating frame.
* Remaining characters: Successive rotation axis `x`, `y` or `z`.

We should include that documentation in this class too.